doc/source/release/1.14.0-notes.rst

```diff
@@ -49,49 +67,119 @@ New features
+  is an interface for COBYQA (Constrained Optimization BY Quadratic
+  Approximations), a derivative-free optimization solver, designed to
+  supersede COBYLA, developed by the Department of Applied Mathematics, The
+  Hong Kong Polytechnic University.
+- There are some performance improvements in
+  `scipy.optimize.differential_evolution`.
+- `scipy.optimize.approx_fprime` now has linear space complexity.
 
 
 `scipy.signal` improvements
 ===========================
+- `scipy.signal.minimum_phase` has a new argument ``half``, allowing the
+  provision of a filter of the same length as the linear-phase FIR filter
+  coefficients and with the same magnitude spectrum.
 
 
 `scipy.sparse` improvements
 ===========================
-
+- A special case has been added to handle multiplying a ``dia_array`` by a
+  scalar, which avoids a potentially costly conversion to CSR format.
+- `scipy.sparse.csgraph.yen` has been added, allowing usage of Yen's K-Shortest
+  Paths algorithm on a directed on undirected graph.
+- Addition between DIA-format sparse arrays and matrices is now faster.
 
 
 `scipy.spatial` improvements
 ============================
-
+- ``Rotation`` supports an alternative "scalar-first" convention of quaternion
+  component ordering. It is available via the keyword argument ``scalar_first``
+  of ``from_quat`` and ``as_quat`` methods.
+- Some minor performance improvements for inverting of ``Rotation`` objects.
 
 `scipy.special` improvements
 ============================
+- Added `scipy.special.log_wright_bessel`, for calculation of the logarithm of
+  Wright's Bessel function.
+- Generalized ufunc support has been expanded.
+- The relative error in `scipy.special.hyp2f1` calculations has improved
+  substantially.
+- Improved behavior of ``boxcox``, ``inv_boxcox``, ``boxcox1p``, and
+  ``inv_boxcox1p`` by preventing premature overflow.
 
 
 `scipy.stats` improvements
 ==========================
-
-Hypothesis Tests
-----------------
-
-
-Sample statistics
------------------
-
-
-Statistical Distributions
--------------------------
-
-
-Other
------
+- A new function `scipy.stats.power` can be used for simulating the power
+  of a hypothesis test with respect to a specified alternative.
+- The Irwin-Hall (AKA Uniform Sum) distribution has been added as
+  `scipy.stats.irwinhall`.
+- Exact p-value calculations of `scipy.stats.mannwhitneyu` are much faster
+  and use less memory.
+- `scipy.stats.pearsonr` now accepts n-D arrays and computes the statistic
+  along a specified ``axis``.
+- `scipy.stats.kstat`, `scipy.stats.kstatvar`, and `scipy.stats.bartlett`
+  are faster at performing calculations along an axis of a large n-D array.
 
 
+**************************
+Array API Standard Support
+**************************
+*Experimental* support for array libraries other than NumPy has been added to
+existing sub-packages in a recent version of SciPy. Please consider testing
```

Comment:
Oops, I see that I had a typo in my PR and wrote "sub-packages in recent version of SciPy". It was supposed to be:
```suggestion
existing sub-packages in recent versions of SciPy. Please consider testing
```
because the work is ongling.